### PR TITLE
add transcript for episode 039

### PR DESCRIPTION
**Pull request by ericseppanen**

### CODE REVIEW

File: _transcripts/039-rust-1.54-1.55.md [hunk 1]
```diff
@@ -0,0 +1,718 @@
...
+
+__Jon__: I feel like we say this, and then we always end up spending, like, a
+ton of time on some really weird detail.
+
+__Ben__: No, I promise. This time it is a pretty small release.
+
+__Jon__: All right, let's try. So, 1.54 starts out with "attributes can invoke
+function-like macros." And this one is, on the surface, pretty simple. If you
+have an attribute macro, so this is something that starts with a square bracket
+and then an optional exclamation mark and then square brackets. If you have one
```
Comment (ericseppanen) [inline _transcripts/039-rust-1.54-1.55.md:33]: I'm tempted to add a correction here:
> ... starts with a square bracket *(editor's note: starts with a `#`)*

(at 01:04)

File: _transcripts/039-rust-1.54-1.55.md [hunk 2]
```diff
@@ -0,0 +1,718 @@
...
+__Ben__: Well, a great way to make things better is to make them worse and then
+they get better.
+
+__Jon__: We're actually almost at the end of this release, so I guess you were
+right that it's fairly short. Although we do have some stabilized APIs and some
+some more detailed changelog items to go through. For stabilized APIs we already
+talked about the wasm32 intrinsics, which have gotten their own module under
+arch called `wasm32`. There's some binary search things added to `VeqDeque`— or
+*vec-deque*, I don't know how it's pronounced— that we're not really going to
+talk about; they're not super interesting.
```
Comment (ericseppanen) [inline _transcripts/039-rust-1.54-1.55.md:165]: I'm always a bit unsure how to transcribe debates about how something is pronounced...

File: _transcripts/039-rust-1.54-1.55.md [hunk 3]
```diff
@@ -0,0 +1,718 @@
...
+
+__Jon__: Yeah, and part of the hope too, I assume, is that because you get to
+see it in your dependencies, you can then go and try to fix them. Like you can
+submit either an issue or a PR to your sort of upstream dependencies and go,
+hey, I was told about this lint from Cargo, here's a thing that just fixes it so
+that your code won't break in the future.
+
+__Ben__: One more little thing that I noticed— not in the release notes, but
+secretly, is that 1.54 has turned on the `mutable_noalias` flag once again, by
+default, on certain LLVM versions. And so we've maybe spoken about this before
```
Comment (ericseppanen) [inline _transcripts/039-rust-1.54-1.55.md:243]: I'm not certain whether this is "mutable noalias" or `mutable_noalias`.

(14:20)

File: _transcripts/039-rust-1.54-1.55.md [hunk 4]
```diff
@@ -0,0 +1,718 @@
...
+like, determine whether they can use a nightly feature or not. And so they would
+break if you had Rust flags that they didn't take into account. That's passed
+in, which is kind of nice.
+
+Another one is that we now have `cargo clippy --fix` stabilized. So we've had
+`cargo fix` for a while, for things like edition changes. But now— or even just
+like if the compiler can automatically fix a given warning or error, I think you
+can pass, like `cargo --fix` or just `cargo fix`, and now we have the same for
+Clippy. So if Clippy detects a lint that it thinks it can— it has like, an
+automated fix for, you can just call `cargo clippy --fix` and it will fix those
```
Comment (ericseppanen) [inline _transcripts/039-rust-1.54-1.55.md:597]: So `cargo --fix` is wrong, and `cargo fix` is correct? I'm not sure how to format this; I could do something like:

> like ~~"cargo dash-dash-fix"~~ or just `cargo fix`

Or do the heavy-handed thing and just delete the wrong one entirely?

(38:55)

File: _transcripts/039-rust-1.54-1.55.md (hunk 1 above)
Comment (jonhoo) [reply _transcripts/039-rust-1.54-1.55.md:33]: Seems reasonable to me!

File: _transcripts/039-rust-1.54-1.55.md (hunk 2 above)
Comment (jonhoo) [reply _transcripts/039-rust-1.54-1.55.md:165]: Hehe, yeah, I don't have a good answer for you. I suppose we could use the phonetic alphabet (IPA) for this to describe the sounds, but that _feels_ like too much work.

Comment (jonhoo) [reply _transcripts/039-rust-1.54-1.55.md:165]: Alternatively write "pronounced like Vec Deck" and "pronounced like Vec D Q"?

File: _transcripts/039-rust-1.54-1.55.md (hunk 3 above)
Comment (jonhoo) [reply _transcripts/039-rust-1.54-1.55.md:243]: `mutable-noalias` technically (https://github.com/rust-lang/rust/pull/88325), but "mutable noalias" works well I think.

File: _transcripts/039-rust-1.54-1.55.md (hunk 4 above)
Comment (jonhoo) [reply _transcripts/039-rust-1.54-1.55.md:597]: I like dash-through for this

File: _transcripts/039-rust-1.54-1.55.md (hunk 2 above)
Comment (ericseppanen) [reply _transcripts/039-rust-1.54-1.55.md:166]: I'm not in love with any of the alternatives, so I just left it as-is?

**Comment by ericseppanen** [comment]
Updated; ready to merge I think.

**Review by jonhoo — APPROVED** [review]

**Comment by jonhoo** [comment]
Thanks!

**Pull request merged**